oaknational/Oak-Web-Application
# Feat/lesq 129/restyle listing components

## Description

**Job Story:**

For both the lesson listing and the unit listing components

## Acceptance Criteria:

https://docs.google.com/spreadsheets/d/17gz87A49Rbic3LHjukrk0l_EBfT17M56vT3YCF7ir0M/edit?usp=sharing

- [x]  Number in a box on the left hand side, no longer in the title
- [x]  Unavailable styling changes
- [x]  Add the lil chevron
- [x]  Make sure they are both the same width on the page
- [x]  Make sure designs for mobile are met
- [x]  No change to amount of lessons per page (Monika is just lazy and didn’t show all 20 in figma)
- [x]  aria-label → combine number and lesson title to give additional context to screen reader users

## Issue(s)

## How to test

1. Go to https://deploy-preview-1768--oak-web-application.netlify.thenational.academy

https://deploy-preview-1768--oak-web-application.netlify.thenational.academy/beta/teachers/programmes/computing-secondary-ks3/units

3. Click on _______
4. You should see _______

## Screenshots

![image](https://github.com/oaknational/Oak-Web-Application/assets/1030540/c6e2bfb9-811a-4030-9b12-ecebbb6268eb)

How it should now look:

![image](https://github.com/oaknational/Oak-Web-Application/assets/1030540/daf2e029-da02-4a84-88ed-3c3aa0f0fef4)

## Checklist
